src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-022.xml.sarif

```diff
@@ -12,6 +12,46 @@
           "mimeType": "text/x-cpp"
         }
       },
+      "logicalLocations": {
+        "f": {
+          "decoratedName": "?f@@YAXXZ"
+        },
+        "h": {
+          "decoratedName": "?h@@YAXPAHH@Z"
+        },
+        "foo": {
+          "decoratedName": "?foo@@YAXPAHI@Z"
+        },
+        "StrLen": {
+          "decoratedName": "?StrLen@@YAIPBD@Z"
+        },
+        "BugLast": {
+          "decoratedName": "?BugLast@@YADPBD@Z"
+        },
+        "g1": {
+          "decoratedName": "?g1@@YAXPAH@Z"
+        },
+        "foo-0": {
+          "name": "foo",
+          "fullyQualifiedName": "foo",
+          "decoratedName": "?foo@@YADXZ"
+        },
```

Comment:
Perfect. There really are two different `foo`s (different decorated names), and this one has the extra properties it needs.